### ISSUE TITLE
PlayerCentered IScores array size 1

### ISSUE THREAD

**Issue by FlyHighBye** [playgameservices/play-games-plugin-for-unity#1682]
When calling the following method, the returned LeaderboardScoreData object **data** is returned with only one value in the IScores array.

`PlayGamesPlatform.Instance.LoadScores (
			leaderboardID,
			LeaderboardStart.PlayerCentered,
			20,
			LeaderboardCollection.Public,
			LeaderboardTimeSpan.AllTime,
			(data) => {
				}
		);`

The returned value is only the user's score. But when calling it with the **TopScores** parameter, I get the correct number of responses for the IScores array.

I am on this version - GooglePlayGamesPlugin-0.9.36

**Comment by arunkarnann** [comment]
Yes , In an other issue they mentioned this Google + circle is no more integrated with Play games [https://android-developers.googleblog.com/2016/12/games-authentication-adopting-google.html](url) . But i am wondering how it is working in default. because for me its showing player circle scores in default Ui, But when i try to load score its not working. Even i tried loading top 50 scores and checked with `user.isFriends` method. Its returns **True** for all users. I hope they fix the issue

**Comment by xoidberg** [comment]
Hey, I have the exact same problem. I'm trying to get the next best scores relative to the current user's score. I'm using the same code as you, but I also get only the user's score in the score array and not of other players. TopScores is working fine.
Did you by any chance find a fix or a workaround for this?
I'm using GooglePlayGamesPlugin-0.9.41

**Comment by FlyHighBye** [comment]
No luck, had to skip this implementation. But since then I have moved on to using Cloud Once [https://github.com/jizc/CloudOnce] for implementing Game Services. Never checked for this issue, but in general, implementation has been an ease.
